src/main/java/io/mosip/print/constant/APIAuthorityList.java

```diff
@@ -0,0 +1,42 @@
+package io.mosip.print.constant;
+
+public enum APIAuthorityList {
+
+	PACKETRECEIVER(new String[] { "REGISTRATION_ADMIN", "REGISTRATION_PROCESSOR", "REGISTRATION_OFFICER",

```

Comment:
Please remove unused files and unused data within the file which you used,then we can merge.